.circleci/config.yml

```diff
@@ -12,3 +45,19 @@ workflows:
           context:
             - snyk-bot-slack
           channel: group-cloud-security-vulnerabilities-alerts
+      - scan-code:
+          name: Scan Code
+          context:
+            - tiki-snyk
```

Comment:
We can use any other context that has a SNYK_TOKEN that can access our org, I just.. don't know of any. This one is used for the k8s-connector repos.